src/control-plane/backend/click-stream-api.ts

```diff
@@ -218,6 +218,23 @@ export class ClickStreamApiConstruct extends Construct {
     });
     stackActionStateMachine.stateMachine.grantStartExecution(stackWorkflowStateMachine.stackWorkflowMachine);
 
+    const quickSigthEmbedRole = new iam.Role(this, 'QuickSigthEmbedRole', {
+      assumedBy: new iam.ServicePrincipal('lambda.amazonaws.com'),
+      inlinePolicies: {
+        quickSightEmbedPolicy: new iam.PolicyDocument({
+          statements: [
+            new iam.PolicyStatement({
+              effect: iam.Effect.ALLOW,
+              resources: ['*'],
```

Comment:
Could we use prefix or tags(in condition) for dashboards managed by the solution?